<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>File</class>
 <widget class="QWidget" name="File">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTreeView" name="FileTreeView"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="UpLevelButton">
            <property name="text">
             <string>Uplevel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CurrentPathEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="FileDetailView"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/client.qrc"/>
 </resources>
 <connections/>
</ui>
